spec/helpers/lib/resourceHints.js

```diff
@@ -0,0 +1,140 @@
+const _ = require('lodash');
+const Code = require('code'),
+    expect = Code.expect;
+const Sinon = require('sinon');
+const Lab = require('lab'),
+    lab = exports.lab = Lab.script(),
+    describe = lab.experiment,
+    it = lab.it,
+    beforeEach = lab.beforeEach,
+    afterEach = lab.afterEach;
+const ResourceHints = require('../../../helpers/lib/resourceHints');
+
+describe('resource hints', function () {
+
+    describe('addResourceHint', function () {
+
+        let sandbox;
+
+        beforeEach(done => {
+            sandbox = Sinon.createSandbox();
+            done();
+        });
+
+        afterEach(done => {
+            sandbox.restore();
+            done();
+        });
+
+        it("does work as expected with valid params", (done) => {
+
+            let globals = {resourceHints: []};
```

Comment:
it's `const` and I think all above variables.
The main rule is if the variable is not reassigned (overwritten), so it's `const`.